subprojects/docs/src/docs/release/notes.md

```diff
@@ -56,6 +56,16 @@ The [PMD](userguide/pmd_plugin.html) and [CodeNarc](userguide/pmd_plugin.html) p
 
 In Java projects, these tools will use the same version of Java required by the project. In other types of projects, they will use the same version of Java that is used by the Gradle daemon.
 
+<a name="configuration"></a>
```

Comment:
Similar to https://github.com/gradle/gradle/blob/release/subprojects/docs/src/docs/release/notes.md#extended-configuration-cache-task-compatibility (in `release`/7.6)